save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/DemoDto.kt

```diff
@@ -24,6 +24,12 @@ data class DemoDto(
     val configName: String? = null,
     val githubProjectCoordinates: ProjectCoordinates? = null,
 ) {
+    /**
+     * @return true of [DemoDto] is valid, false otherwise
+     */
+    @Suppress("FUNCTION_BOOLEAN_PREFIX")
+    fun validate(): Boolean = !projectCoordinates.consideredBlank() && runCommand.isNotBlank() && fileName.isNotBlank()
```

Comment:
isValid? and remove `Suppress`?

Reply:
Problem with Jackson serialization - isValid will be serialized (dnw)

Reply:
> Problem with Jackson serialization

1. You can register a mix-in class and add a `@JsonIgnore`-annotated property &mdash; see `TestMetricsMixin`.
2. You can annotate a properly-named function with an unusual `@JvmName` and have Jackson effectively ignoring it, even w/o any mix-in.